src/sentry/static/sentry/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -103,6 +104,37 @@ export function getOnboardingTasks(
       location: `/organizations/${organization.slug}/projects/new/`,
       display: true,
     },
+    {
+      task: OnboardingTaskKey.FIRST_TRANSACTION,
+      title: t('Enable Performance'),
+      description: t(`See what's slow`),
+      detailedDescription: t(
+        `Add Performance to track latency and bottlenecks within your application.
+        Monitor mission-critical endpoints and set metric alerts to proactively address the worst-performing areas.`
+      ),
+      skippable: true,
+      requisites: [OnboardingTaskKey.FIRST_PROJECT, OnboardingTaskKey.FIRST_EVENT],
```

Comment:
Wouldn't it be possible to send a transaction before your first error? Should this only depend on the project?

Reply:
It is, I wasn't sure if we wanted to fork the onboarding flow at that point or if we wanted people to continue with adding their first project and event and then going down the road of transactions.

I like the idea of having it along side sending your first error but wasn't sure if that needed to go through a longer review process?